data.py

```diff
@@ -36,6 +36,14 @@
         sy.Map({
             "name": sy.Str(),
             "description": sy.Str(),
+            sy.Optional("category"): sy.CommaSeparated(sy.Enum([
```

Comment:
Why make it optional? Shouldn't all projects belong to a category? (In particular since there is the "catch-all" _Other_)